test_runner/src/test/kotlin/Debug.kt

```diff
@@ -3,8 +3,10 @@
 import ftl.Main
 import ftl.run.exception.withGlobalExceptionHandling
 import picocli.CommandLine
+import java.io.File
 
 fun main() {
+    println(File("").absolutePath)
```

Comment:
do we need this?

Reply:
Can be useful to determine if execution dir is a repository root or `test_runner` dir

Reply:
Maybe append "Running path: " at start of println message

Reply:
`Debug.kt` is only for debugging purpose

Reply:
```suggestion
    println("Running path: ${File("").absolutePath}")
```